source-hubspot-native/source_hubspot_native/models.py

```diff
@@ -139,7 +145,7 @@ class Owner(BaseDocument, extra="allow"):
 class BaseCRMObject(BaseDocument, extra="forbid"):
     ASSOCIATED_ENTITIES: ClassVar[list[str]]
 
-    class History(BaseDocument, extra="forbid"):
+    class History(BaseModel, extra="forbid"):
```

Comment:
This should have always been `BaseModel` instead of `BaseDocument`, since `BaseDocument` includes top-level document fields like `_meta/op`. More significantly to the discovery snapshot is that the `propertiesWithHistory` won't be output in the "minimal" schema at all, since in the majority of cases users won't capture property history so it should be supressed from the schema.